vite.config.ts

```diff
@@ -135,6 +135,14 @@ export default defineConfig(async () => {
             // functionality which depends on libxmljs/libxslt.
             threads: false,
 
+            // For future reference: Vitest version 0.29.0+
+            // would require the `singleThread` setting instead
+            // of `threads`. We decided not to update past
+            // 0.28.5 in the 3.0.1 release, as newer releases
+            // also caused problems with changes to the
+            // snapshot format.
+            // singleThread: true,
+
```

Comment:
This isn't really related. But it took a fair bit of time to figure out, and I'd hate to repeat that if/when we update Vitest again in the future.